ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/AUTO_BGP_ASN.yml

```diff
@@ -0,0 +1,58 @@
+type: l3leaf
+mgmt_gateway: 192.168.202.1
+
+l3leaf:
+  defaults:
+    platform: vEOS-LAB
+    loopback_ipv4_pool: 192.168.255.0/24
+    loopback_ipv4_offset: 2
+    vtep_loopback_ipv4_pool: 192.168.254.0/24
+    uplink_interfaces: []
+    uplink_switches: []
+    uplink_ipv4_pool: 172.31.255.0/24
+    bgp_defaults: ['no bgp default ipv4-unicast', 'distance bgp 20 200 200']
+    mlag_interfaces: [ Ethernet3, Ethernet4 ]
+    virtual_router_mac_address: 00:dc:00:00:00:0a
+    mlag_peer_l3_ipv4_pool: 10.255.251.0/24
+    mlag_peer_ipv4_pool: 10.255.252.0/24
+    spanning_tree_mode: mstp
+    spanning_tree_priority: 4096
+    bgp_as: 65101-65110
+  node_groups:
```

Comment:
I am 99.9% sure it works already given that it has a `switch.id` but maybe lets add a test with a `nodes` not inside a `node_groups`

Reply:
Good catch, will do.

Reply:
Done, works great!